packages/web/nexus/user.ts

```diff
@@ -347,7 +347,7 @@ const UserMutations = extendType({
         const isValid = await bcrypt.compare(args.oldPassword, user.auth.password)
 
         if (!isValid) {
-          throw new Error('Invalid old password')
+          throw new Error('Old password is incorrect')
```

Comment:
@Lanny any thoughts on how we want to handle translations for errors occurring server-side that are user-facing?

Reply:
We have some custom errors we can throw that can have attributes like an error code. Mapping error codes to messages on the client is usually the best way to do this. So instead of throwing `Error` make a new error like `InvalidPasswordError` or something in `nexus/errors.ts` that has an error code and we can watch for that on the client.